ci-operator/jobs/openshift/origin/openshift-origin-master-presubmits.yaml

```diff
@@ -398,6 +398,70 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )e2e-aws-image-registry,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - master
+    context: ci/prow/e2e-aws-jenkins
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    name: pull-ci-openshift-origin-master-e2e-aws-jenkins
+    rerun_command: /test e2e-aws-jenkins
```

Comment:
You can manually add a `run_if_changed` value which ensures this test runs only if certain files/directories are modified. Otherwise I think this will be run on every origin PR.

See #4093

Reply:
yep manually setting always_run to false was kept, even after I ran ci-operator-prowgen

Reply:
as I mentioned in the other comment I only want this to run if some triggers the job manually